jxl/src/render/low_memory_pipeline/save/mod.rs

```diff
@@ -33,7 +33,12 @@ impl SaveStage {
         full_image_size: (usize, usize),
         frame_origin: (isize, isize),
     ) -> Result<()> {
-        let Some(buf) = buffers[self.output_buffer_index].as_mut() else {
+        // Skip if buffer index is out of range (e.g., for reference frame buffers
```

Comment:
Are there valid cases in which the index is out of range? IIRC this should only happen if there are logic errors in the decoder.

Reply:
had issues with the newton one! 8bit animation, will test at the end of PR